types/aws-iot-device-sdk/index.d.ts

```diff
@@ -194,10 +194,9 @@ export class device extends EventEmitter {
    * Unsubscribe from a topic or topics
    *
    * @param topic  is a String topic or an array of topics to unsubscribe from
-   * @param options
    * @param callback  fired on unsuback
```

Comment:
If the `callback` is optional, then update JSDoc
```suggestion
   * @param [callback]  fired on unsubscribe from a topic or topics
```